README.md

```diff
@@ -132,7 +132,7 @@ Data which is stored upon startup includes:
 
 * Now, you can press the Start button in Visual Studio to run the server (it is a green play button in the top middle of the tool bar). It will open the web browser and, after a period that should just be a few minutes long but sometimes lasts half an hour or more, display an Error 403.14 - Forbidden. This is expected. You can now begin manually testing the API.
 
-* If you haven't already, do an `npm install`. This only needs to be done once, then you can just `npm start`
+* If you haven't already, you'll need to `npm install` in the gordon-360-ui-repo. This only needs to be done once, then you can just `npm start`.
```

Comment:
There should not be a dash connecting the repo name and "repo".  Should be "gordon-360-ui repo"